prodigy.el

```diff
@@ -103,6 +103,23 @@ An example is restarting a service."
 (defvar prodigy-view-confirm-clear-buffer t
   "`prodigy-view-clear-buffer' will require confirmation if non-nil.")
 
+
+(defvar prodigy-dependency-services nil
```

Comment:
I would greatly prefer if these were not variables but functions which recompute every time from `prodigy-services`.  Unless you have 100000 services a topo-sort is extremly fast and only having a single source of truth makes things a lot simpler.